java11/src/Arrays.java

```diff
@@ -0,0 +1,16 @@
+public class Arrays {
+    public static void main(String[] args) {
+        int number = 10;
+        int[] numbers = new int[5];
+        for (int i = 0; i < numbers.length;i++){
+            numbers[i]=i*10;
```

Comment:
Thats a little thing but anyway its kinda git flow practice, so you should add spaces before and after '=' and '*'. It helps with readability. 

numbers[i] = i * 10;